README.md

```diff
@@ -31,8 +31,10 @@ from qiskit import Aer, IBMQ  # import the Aer and IBMQ providers
 from qiskit.providers.aer import noise  # import Aer noise models
 
 # Choose a real device to simulate
-IBMQ.load_accounts()
-device = IBMQ.get_backend('ibmq_16_melbourne')
+token = 'your-ibm-account-token-here'
+IBMQ.save_account(token)
```

Comment:
I don't think we need to have the save token example here, just loading a saved account. Can you change to:

```suggestion
provider = IBMQ.load_account()
device = provider.get_backend('ibmq_ourense')
```